security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -245,7 +254,8 @@ func (sc *SecretCache) GenerateSecret(ctx context.Context, connectionID, resourc
 			return nil, err
 		}
 		// This is not stored - envoy will refresh when the cert is about to expire.
-		cacheLog.Infoa("GenerateSecret from file", resourceName)
+		cacheLog.Infoa("GenerateSecret from file ", resourceName)
+		sc.secrets.Store(connKey, *ns)
```

Comment:
nit - the comment says it is not stored but we store it

Reply:
Nice catch! Thanks! This comment is not correct, I have updated it.